### ISSUE TITLE
Rebooting of nodes doesn't seem to be working

### ISSUE THREAD

**Issue by skynet01** [Odianosen25/Monitor-App#73]
I see in the MQTT log that it tries to reboot my nodes when they are unavailable howe they never reboot. Is there a way to debug this? I tried running "sudo /sbin/reboot now" on the node manually in the shell and it works perfectly. I don't use ssh keys and use password login for them.

Maybe the string in the config needs to have quotes around it?

Thanks for your help as always!

**Comment by Odianosen25** [comment]
Hello @skynet01,

Apologies really been a long while and seems I didn't get your alert.

> Maybe the string in the config needs to have quotes around it?

No it shouldn't need to be in quotes actually. After you notice it tries to reboot it, are there by any chance any follow up errors? If it can't reach the node,  then it should error out.

How do you nice its not rebooting?  via running time?

**Comment by skynet01** [comment]
Yeah, the running time says 157 days. The log in mqt says it's sending a reboot command but I guess nothing happens.

**Comment by Odianosen25** [comment]
Hello @skynet01,

Can you kindly run the app in debug mode? If it runs, I have the result being sent to the logs in debug mode. That way you will know if it’s running.

Also are you using dev or master?

regards
